pkg/cloud/gcp/actuators/services/tags/tagservice_mock.go

```diff
@@ -0,0 +1,26 @@
+package tagservice
+
+import (
+	"context"
+
+	tags "google.golang.org/api/cloudresourcemanager/v3"
+)
+
+type MockTagService struct {
```

Comment:
Exported types and functions should have godocs

Reply:
suggestion incorporated.